array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-unused-vars */
+function tail(array) {
+  var afterFirst = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== 0) {
```

Comment:
Instead of needing a conditional here, can you set this up where you don't even need to have an iteration for `0`?

Reply:
Updated.

Reply:
I don't see an update